docker-compose.yml

```diff
@@ -0,0 +1,104 @@
+version: '3.4'
+
+# This is an example docker-compose file for IPFS Cluster
+# It runs two Cluster peers (cluster0, cluster1) attached to two
+# IPFS daemons (ipfs0, ipfs1).
+#
+# It expects a "compose" subfolder as follows where it will store configurations
+# and states permanently:
+#
+# compose/
+# |-- cluster0
+# |-- cluster1
+# |-- ipfs0
+# |-- ipfs1
+#
+# 
+# During the first start, default configurations are created for all peers.
+
+services:
+
+##################################################################################
+##################################################################################
+## Cluster PEER 0 ################################################################
+##################################################################################
+
+  ipfs0:
+    container_name: ipfs0
+    image: ipfs/go-ipfs:release
+    ports:
+          - "4001:4001" # ipfs swarm
+          - "4002:4002/udp" # udp swarm
+#         - "5001:5001" # expose if needed/wanted
+#         - "8080:8080" # exposes if needed/wanted
+    volumes:
+      - ./compose/ipfs0:/data/ipfs
+      
+  cluster0:
+    container_name: cluster0
+    image: ipfs/ipfs-cluster:latest
+    depends_on:
+      - ipfs0
+    environment:
+      # Change this! Same for all peers!
+      CLUSTER_SECRET: adc72159e0ce63350c13de6a5e4d3b2f500232a770cb8c3e7829fc514d89d109 
```

Comment:
Wouldn't it be better to set this to an environment variable and put a 'default' cluster secret in the Makefile?

Reply:
Yes... it would be better. I just don't want people to forget and this is an example. But I guess a cluster without secret is as bad as a cluster with a default one. At least we can generate one on the fly in the Makefile. I'll fix this.